.github/workflows/benchmark.yml

```diff
@@ -41,34 +41,53 @@ jobs:
       - name: checkout
         uses: actions/checkout@v3
 
-      # TODO: remove "--no-check-certificate" when possible
-      - name: Download pre-generated indices
-        timeout-minutes: 60
-        run: wget --no-check-certificate -q -i tests/benchmark/data/hnsw_indices.txt -P tests/benchmark/data
+      - name: Print runner info
+        run: |
+          printf "Runner lscpu:\n$(lscpu)\n"
+          printf "Runner lsmem:\n$(lsmem)\n"
+          printf "Runner nproc:\n$(nproc)\n"
+          printf "Runner uname:\n$(uname -a)\n"
 
       - name: Install benchmark dependencies
         run: |
-             sudo apt update
+             sudo .install/install_script.sh
              sudo apt install python3-pip -y
-             sudo pip3 uninstall virtualenv
-             sudo pip3 install --upgrade pip PyYAML
+             pip3 install --upgrade pip PyYAML setuptools
              pip3 install redisbench-admin
+             pip3 install -r requirements.txt
+
+      - name: stress test
+        run: |
+          sudo apt install stress-ng -qqy
+          uptime
+          stress-ng -c 1 --timeout 60s --metrics-brief
+          uptime
+          stress-ng --stream 1 -t 60 --metrics-brief
+          uptime
+          stress-ng --ipsec-mb=1 -t 60 --metrics-brief
+          uptime
```

Comment:
do we still need this?

Reply:
Yes, Filipe asked me to keep this and he will add a time series with this data